guides/common/modules/proc_importing-individual-iso-images-and-files.adoc

```diff
@@ -8,14 +8,14 @@ However, if the size of the file that you want to upload is larger than 15 MB, y
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-importing-individual-iso-images-and-files[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Products*, and in the Products window, click *Create Product*.
+. In the {ProjectWebUI}, navigate to *Content* > *Products*, and in the products window, click *Create Product*.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Products*, and click *Create Product*.
```
We can probably remove this, because the user is clearly navigated.